docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,158 @@
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__, **Login**, and go to **EpiCoV** > **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+2. Select a custom set of records. As an example, this guide will look for all records that pass the following criteria:
+
+   1. From Idaho
+   2. Collected between ``2022-02-15`` and ``2022-03-15``
+   3. Has a complete genome
+   4. Has an exact collection date
+
+   .. image:: ../images/gisaid-select-records-idaho-highlighted.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You can choose your own focal set here, but note that more records will take the workflow longer to run, and the GISAID web interface has a maximum of 5,000 records per download.
+
+      Due to the subsampling logic defined in the pre-written ``--configfile``, this tutorial works best when the focal set is geographically within in the USA.
+
+3. Select **Download** > **Input for the Augur pipeline** > **Download**.
+4. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+5.  Move these extracted files to the ``data/`` directory in the ``ncov/`` directory.
+6.  Rename the files as appropriate, for example ``idaho.metadata.tsv`` and ``idaho.sequences.fasta``.
+
+Edit the workflow configuration file
+------------------------------------
+
+This time, you will make some modifications to the pre-written configuration file ``genomic-surveillance.yaml`` before running the workflow. The contents are copied below (comments excluded):
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       aligned: https://data.nextstrain.org/files/ncov/open/reference/aligned.fasta.xz
+     - name: custom_data
+       metadata: data/idaho.metadata.tsv
+       sequences: data/idaho.sequences.fasta
+     - name: background_data
+       metadata: https://data.nextstrain.org/files/ncov/open/north-america/metadata.tsv.xz
+       aligned: https://data.nextstrain.org/files/ncov/open/north-america/aligned.fasta.xz
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     idaho:
+       title: "Idaho-specific genomic surveillance build"
+       subsampling_scheme: idaho_scheme
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+   subsampling:
+     idaho_scheme:
+       custom_sample:
+         query: --query "(custom_data == 'yes')"
+         max_sequences: 5000
+       usa_context:
+         query: --query "(background_data != 'yes') & (country == 'USA')"
+         max_sequences: 1000
+         group_by: division year month
+         priorities:
+           type: proximity
+           focus: custom_sample
+       global_context:
+         query: --query "(reference_data == 'yes')"
```

Comment:
Mostly a note-to-self to edit in these queries `ncov-tutorial` repo and in this branch so we're querying contextual data from both background and reference data. We'll want the `usa_context` query to be `query: --query "(custom_data != 'yes') & (country == 'USA')"` and the `global_context` query to be `query: --query "(custom_data != 'yes')"`.

The current queries will end up excluding the background data from all of the subsampling rules.

Reply:
Fixed: 4e5894f6 and https://github.com/nextstrain/ncov-tutorial/commit/57a8d91021396ab72d077a5041b28a06d6abfa0e